docs/examples/install_tendermint.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# get and unpack golang
+curl -O https://storage.googleapis.com/golang/go1.9.2.linux-amd64.tar.gz
+tar -xvf go1.9.2.linux-amd64.tar.gz
+
+apt install make
+
+## move go and add binary to path
+mv go /usr/local
+echo "export PATH=\$PATH:/usr/local/go/bin" >> ~/.profile
+
+## create the GOPATH directory, set GOPATH and put on PATH
+mkdir goApps
```

Comment:
is this from official guides?

Reply:
the official guide doesn't even set a `$GOPATH`, so no. AFAICT there is no standard for naming this directory (I've seen `work`, `gocode`, `go, `goApps` etc, the list goes on)